packages/api/src/sequelize/migrations/2023-07-27_00_alter-facility-remove-facilityNumber.ts

```diff
@@ -0,0 +1,23 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+
+const tableName = "facility";
+const columnName = "facility_number";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.removeColumn(tableName, columnName, { transaction });
+  });
+};
+
+export const down: Migration = ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.addColumn(
+      tableName,
+      columnName,
+      { type: DataTypes.INTEGER },
+      { transaction }
+    );
+  });
+};
```

Comment:
In the case of rollback, will this column now be empty and lose the numbers?

Reply:
Yes. That's why the plan is to only merge this a week or two after the release is done (the date became outdated b/c we're taking long to deploy the original migrations).